### ISSUE TITLE
aurelia-cli tile potentially confusing

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/global-issues#18]
I When clicked on,  and then clicked on `Select aurelia.json it renders this:

![image](https://cloud.githubusercontent.com/assets/2712405/17686326/94966964-6339-11e6-8492-2f94d0e6e63a.png)

<br>

So, you are supposed to know where this file is (it should be Monterey that knows that).

I navigated to aurelia_project and selected the `aurelia.json` file - and got this as the result
<br>

![image](https://cloud.githubusercontent.com/assets/2712405/17686356/d87b9000-6339-11e6-9d87-a1dcc1ef95c8.png)

<br>

I believe that this is a bug as I also see a warning in the footer (not sure when it got raises, sorry - perhaps this is a chance to look at the log).

![image](https://cloud.githubusercontent.com/assets/2712405/17686382/05f6973c-633a-11e6-93fd-62ec998d9222.png)

**Comment by JeroenVinke** [comment]
need more info. Did tou try and configure a aurelia.json path for a project that is not an aurlelia-cli project? Because it has always detected it for me without problems. Was this maybe caused by the confusion over https://github.com/monterey-framework/global-issues/issues/17#issuecomment-240010032 ?

**Comment by JeroenVinke** [comment]
If so, should we change the text that says "show irrelevant tiles" ?

**Comment by adriatic** [comment]
Nothing to do with "irrelevant tiles" - you could follow the complete "path" of my first (and only) session here https://github.com/aurelia-ui-toolkits/assets/issues/4

I did not deviate from this path and the few error reports are made stem from this exercise. Hope it is sufficient information.

**Comment by JeroenVinke** [comment]
in the background of [this screenshot](https://cloud.githubusercontent.com/assets/2712405/17685823/80917714-6335-11e6-8064-24b31d64e99c.png) I see the aurelia-cli tile, meaning that monterey has successfully detected aurelia-cli. I think this means that you trying to select the aurelia.json file ([here](https://cloud.githubusercontent.com/assets/2712405/17686326/94966964-6339-11e6-8492-2f94d0e6e63a.png)) was not necessary as monterey already knew where it was. 

Is this correct? If so, we need to explain this on the screen. What we currently have in the aurelia-cli screen is a kind of placeholder as we will probably add a GUI for all other commands of aurelia-cli

**Comment by adriatic** [comment]
Yes, this is the situation where I lost the memory of what am I doing. Safe to ignore this discussion
